src/main/resources/application.yaml

```diff
@@ -36,14 +35,19 @@ spring:
       maxLifetime: 7200000
       connectionTimeout: 30000
     jpa:
+      hibernate:
+        ddl-auto: none
       database-platform: org.hibernate.dialect.PostgreSQLDialect
       properties:
         hibernate:
+          default_schema: ${spring.datasource.schema}
+          dialect: org.hibernate.dialect.PostgreSQLDialect
           globally_quoted_identifiers: true
           jdbc:
             lob:
               # silence the 'wall-of-text' - unnecessary exception throw about blob types
               non_contextual_creation: true
+            time_zone: UTC
```

Comment:
I don't know much about how timezones are managed in jdbc but do we definitely want UTC? I'm wondering if we just want to store the local time GMT/BST Timezone London and ensure the field types in postgres store the tz alongside the date and time?